core/system/rule_manager.go

```diff
@@ -55,9 +31,28 @@ func GetRules() []*SystemRule {
 }
 
 // LoadRules loads given system rules to the rule manager, while all previous rules will be replaced.
-func LoadRules(rules []*SystemRule) (bool, error) {
-	ruleChan <- rules
-	return true, nil
+func LoadRules(rules []*SystemRule) bool {
+	if len(rules) == 0 {
+		logger.Debug("the rules to be loaded is empty")
+		return true
```

Comment:
It's good to have a `ClearRules` method, but we need to support clear rules by passing an empty slice to `LoadRules` method.

Reply:
It seems that the problem had been fixed.